crates/holochain/src/core/ribosome/host_fn/create.rs

```diff
@@ -352,20 +352,20 @@ pub mod wasm_test {
         let entry: Entry = (&Post("foo".into())).try_into().unwrap();
         let entry_hash = EntryHash::with_data_sync(&entry);
         assert_eq!(
-            "uhCEkPjYXxw4ztKx3wBsxzm-q3Rfoy1bXWbIQohifqC3_HNle3-SO",
+            "uhCEkhZil6EIK4jeFSY6LXh1h3fJvTYgAbU4vxQAsAgVKxwHDlMJU",
```

Comment:
entry hashes are now computed differently since the serialized bytes of the `Entry` enum differ with the new enum format.